rwengine/include/script/ScriptTypes.hpp

```diff
@@ -140,6 +300,52 @@ template<> GameObject* ScriptArguments::getObject<PickupObject>(unsigned int arg
 /** Special player-index returning function */
 template<> GameObject* ScriptArguments::getObject<PlayerController>(unsigned int arg) const;
 
+template<> float ScriptArguments::getParameter<float>(unsigned int arg) const;
+template<> int ScriptArguments::getParameter<int>(unsigned int arg) const;
+template<> char const* ScriptArguments::getParameter<char const*>(unsigned int arg) const;
+template<> glm::vec2 ScriptArguments::getParameter<glm::vec2>(unsigned int arg) const;
+template<> glm::vec3 ScriptArguments::getParameter<glm::vec3>(unsigned int arg) const;
+template<> glm::u8vec3 ScriptArguments::getParameter<glm::u8vec3>(unsigned int arg) const;
+template<> glm::u8vec4 ScriptArguments::getParameter<glm::u8vec4>(unsigned int arg) const;
+
+template<> float& ScriptArguments::getParameterRef<float>(unsigned int arg) const;
+template<> int& ScriptArguments::getParameterRef<int>(unsigned int arg) const;
+
+///////////////////////////////////////////////////////////
+// Script Object specialisations
+template <>
+ScriptObjectType<VehicleObject> ScriptArguments::getScriptObject(
+    unsigned int arg) const;
```

Comment:
Can these arguments go on the same line (it's not 1980 anymore if you're worried about line length.)